docroot/sites/default/settings.php

```diff
@@ -169,6 +169,11 @@
   include $app_root . '/' . $site_path . '/settings/settings.fast_404.php';
 }
 
+// Deploy Settings
```

Comment:
I'd like to see some explanation here like...
```suggestion
// Jenkins moves this file into place during deploy, so if it is present we are in deploy mode.
```